web/src/layout/map.js

```diff
@@ -113,11 +111,10 @@ export default () => {
 
   const handleZoneClick = useMemo(
     () => (id) => {
-      trackEvent('countryClick');
```

Comment:
Why remove this, don't we still want to keep it (but renamed to match conventions?)

Reply:
My thoughts here were that it is firing a lot and a lot of the same insights can be seen by the URLs.

We can keep it if we want and rename either to the same event as the one in ranking or something like ZoneOnMap Clicked

Reply:
Thanks for the response, I agree now 🙃